autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/ui/credential/management/AutofillSettingsViewModel.kt

```diff
@@ -152,6 +156,7 @@ class AutofillSettingsViewModel @Inject constructor(
     private var combineJob: Job? = null
 
     private var currentUrl: String? = null
+    private var privacyProtectionStatus: PrivacyProtectionStatus = Unknown
```

Comment:
ok, this is something similar to `currentUrl`. I think that should belong to the view state, and not having it here separated. Seems connected to the report, currentUrl, as a single thing, and we are keeping them independent which may be prone to errors.

Reply:
As discussed, wrapped up into a single object and stored in the view state